src/components/Button.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './styles/Button.css';
+
+const Button = (props) => {
+  const { buttonName } = props;
+  const { color } = props;
+  const { wide } = props;
+
```

Comment:
you can knock all 03 into ONE line; something like
` const { buttonName, color, wide } = props; `